agent/handlers/types/v1/response.go

```diff
@@ -28,6 +28,7 @@ type TaskResponse struct {
 	Family        string
 	Version       string
 	Containers    []ContainerResponse
+	Networks      []NetworkResponse
```

Comment:
Similar to https://github.com/aws/amazon-ecs-agent/blob/cc9fcff8e6977895cfafc49e04bdb4df3945b55a/agent/handlers/types/v2/response.go#L60, this should be part of the `ContainerResponse` struct.